src/olympia/stats/templates/stats/stats.html

```diff
@@ -180,15 +180,6 @@ <h1>
         </div>
       </div>
     {% endblock %}
-    {% if addon %}
-      <div id="stats-permissions">
-        <p>{{ _('This dashboard is <b>private</b>.') }}</p>
```

Comment:
Should this be left here but moved out of the `{% if ... ` block or is it mentioned somewhere else on the page?

Reply:
It is not mentioned anymore, no. There is no notion of private vs public dashboard anymore. Would you still keep such a sentence? (Or rather why would we mention this here and not on other devhub pages for instance?)

Reply:
Because the dashboard privacy has been a setting for 10+ years so existing developers might make assumptions.  Just asking though.

Reply:
That's a good suggestion. I'll add the sentence back.